data-serving/data-service/test/controllers/preprocessor.test.ts

```diff
@@ -686,4 +689,107 @@ describe('batch update', () => {
             c2.toObject(),
         );
     });
+    describe('batch delete', () => {
+        it('creates case revisions from caseIds', async () => {
+            const c = await new Case(minimalCase).save();
+            const c2 = await new Case(minimalCase).save();
+
+            const requestBody = { caseIds: [c._id, c2._id] };
+            const nextFn = jest.fn();
+            await createBatchDeleteCaseRevisions(
+                { body: requestBody, method: 'DELETE' } as Request,
+                {} as Response,
+                nextFn,
+            );
+
+            expect(nextFn).toHaveBeenCalledTimes(1);
+            expect(await CaseRevision.collection.countDocuments()).toEqual(2);
+            expect((await CaseRevision.find())[0].case.toObject()).toEqual(
+                c.toObject(),
+            );
+            expect((await CaseRevision.find())[1].case.toObject()).toEqual(
+                c2.toObject(),
+            );
+        });
+        it('creates case revisions from query', async () => {
+            // Simulate index creation used in unit tests, in production they are
+            // setup by the setup-db script and such indexes are not present by
+            // default in the in memory mongo spawned by unit tests.
+            await mongoose.connection.collection('cases').createIndex({
+                notes: 'text',
+            });
+
+            const c = new Case(minimalCase);
+            c.notes = 'got it at work';
```

Comment:
Is this necessary? The case objects should differ by `_id`.

Reply:
Nope, relic of copy/paste from other test. Removed.